mlrun/db/httpdb.py

```diff
@@ -1223,8 +1223,11 @@ def delete_function(self, name: str, project: str = ""):
                 background_task.status.state
                 == mlrun.common.schemas.BackgroundTaskState.failed
             ):
+                reason = background_task.status.error
                 logger.info(
-                    "Function deletion failed", project_name=project, function_name=name
+                    f"Function deletion failed. Reason: {reason}",
```

Comment:
I think it is better to include the `reason` as an argument, similar to `project_name`

Reply:
@moranbental make sense! fixed